eu.usdx.UltraStarDeluxe.yaml

```diff
@@ -0,0 +1,194 @@
+  cleanup:
+  - /share/projectM/fonts
+  - /bin
+
+- shared-modules/lua5.4/lua-5.4.json
+
+- name: portmidi
+  sources:
+  - type: archive
+    url: https://sourceforge.net/projects/portmedia/files/portmidi/217/portmidi-src-217.zip
+    sha256: 08e9a892bd80bdb1115213fb72dc29a7bf2ff108b378180586aa65f3cfd42e0f
+  - type: shell
+    commands:
+    - find -type f | while read a ; do tr -d '\r' < "$a" > /tmp/t && cat /tmp/t > "$a" ; done
+  - type: patch
+    paths:
+# Patches taken from Debian portmidi 217-6.1
+    - patches/00_cmake.diff
+    - patches/02_pmlinuxalsa.diff
+    - patches/03_pm_test_Makefile.diff
+    - patches/11-pmlinuxalsa.patch
+    - patches/13-disablejni.patch
+    - patches/20-movetest.diff
+    - patches/21-hardentests.diff
+    - patches/30-porttime_cmake.diff
+    - patches/40-test_sysex.diff
+    - patches/41-pm_linux.diff
+    - patches/50-change_assert.diff
+    - patches/51-remove_assert.diff
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=Release
+  cleanup:
+  - "*.a"
+  - "libporttime.*"
+
+- name: ultrastardx
+  sources:
+  - type: git
+    url: https://github.com/UltraStar-Deluxe/USDX.git
+    tag: v2023.3.0
+    commit: 36ca911387357295d295b6557aae36f574941b8a
+    disable-submodules: true
+  - type: patch
+    paths:
+    - patches/appstream.patch
+  - type: shell
+    commands:
+# We are passed in some linker flags suitable for gcc, but fpc directly calls ld
+    - sed -i '1iCOMMA=,' src/Makefile.in
+    - sed -i 's/$(LDFLAGS)/$(subst $(COMMA), ,$(patsubst -Wl%,%,&))/' src/Makefile.in
+# Icons and desktop file are not installed automatically
+    - sed -i '/^install-data:/a\\t$(INSTALL_DATA) -D dists/ultrastardx.appdata.xml $(datarootdir)/metainfo/eu.usdx.UltraStarDeluxe.metainfo.xml' Makefile.in
+    - sed -i '/^install-data:/a\\t$(INSTALL_DATA) -D dists/ultrastardx.desktop $(datarootdir)/applications/ultrastardx.desktop' Makefile.in
+    - sed -i '/^install-data:/a\\t$(INSTALL_DATA) -D icons/ultrastardx-icon.svg $(datarootdir)/icons/hicolor/scalable/apps/ultrastardx.svg' Makefile.in
+    - sed -i '/^install-data:/a\\t$(INSTALL_DATA) -D icons/ultrastardx-icon_32.png $(datarootdir)/icons/hicolor/32x32/apps/ultrastardx.png' Makefile.in
+    - sed -i '/^install-data:/a\\t$(INSTALL_DATA) -D icons/ultrastardx-icon_256.png $(datarootdir)/icons/hicolor/256x256/apps/ultrastardx.png' Makefile.in
+    - sed -i '/^install-data:/a\\t$(INSTALL_DATA) -D icons/ultrastardx-icon_512.png $(datarootdir)/icons/hicolor/512x512/apps/ultrastardx.png' Makefile.in
+# Workaround for flathub/org.freedesktop.Sdk.Extension.freepascal#10
+    - cp /usr/lib/sdk/freepascal/etc/fpc.cfg fpc.cfg
+    - ln -s ../fpc.cfg src/
+    - "set -- `gcc -print-file-name=crtbegin.o` ; echo -Fl${1%/*} >> fpc.cfg"
+  buildsystem: autotools
+  config-opts:
+  - --with-libprojectM
+  - --with-opencv-cxx-api
+  - --enable-debug
```

Comment:
you enable debug here....

Reply:
this is what buildbot is complaining about.

Reply:
Does not affect optimization by the compiler. Only adds debug symbols (which flatpak-builder splits off into a eu.usdx.UltraStarDeluxe.Debug extension) and writes log messages to the console. I'll remove that line to make flatpak-builder-lint happy.

Reply:
> and writes log messages to the console

So it slow down for no reason.